sql/mdl.cc

```diff
@@ -3996,7 +4006,10 @@ bool MDL_context::visit_subgraph(MDL_wait_for_graph_visitor *gvisitor) {
         fact by setting VICTIM status to its wait slot.
 */
 
-void MDL_context::find_deadlock() {
+void MDL_context::find_deadlock(MDL_request *mdl_request) {
```

Comment:
Azure Pipelines are failing for release builds with below error.
```
[ 48%] Building CXX object unittest/gunit/CMakeFiles/sqlgunitlib.dir/__/__/sql/stateless_allocator.cc.o
[ 48%] Building CXX object unittest/gunit/CMakeFiles/sqlgunitlib.dir/__/__/sql-common/sql_string.cc.o
/home/vsts/work/1/s/sql/mdl.cc:4009:46: error: unused parameter 'mdl_request' [-Werror=unused-parameter]
 void MDL_context::find_deadlock(MDL_request *mdl_request) {
                                              ^
cc1plus: all warnings being treated as errors
unittest/gunit/CMakeFiles/sqlgunitlib.dir/build.make:92: recipe for target 'unittest/gunit/CMakeFiles/sqlgunitlib.dir/__/__/sql/mdl.cc.o' failed
make[2]: *** [unittest/gunit/CMakeFiles/sqlgunitlib.dir/__/__/sql/mdl.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
Scanning dependencies of target ddl_rewriter-t
```
Please place the argument within `#ifndef DBUG_OFF` and `#endif` blocks.